draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1445,9 +1450,10 @@ user agent MUST process the cookie as follows:
 15. If the cookie's `same-site-flag` is not "None":
 
     1.  If the cookie was received from a "non-HTTP" API, and the API was called
-        from a context whose "site for cookies" is not an exact match for
-        request-uri's host's registrable domain, then abort these steps and
-        ignore the newly created cookie entirely.
+        from a browsing context's active document whose "site for cookies" is
+        not same-site with the top-level browsing context's active document's
+        "site for cookies", then abort these steps and ignore the newly created
```

Comment:
Any reason not to say "top-level origin" in place of 'the top-level browsing context's active document's "site for cookies"'?

Reply:
I was told I got paid by word-count.... 

(I can change it!)